src/task.ts

```diff
@@ -57,6 +57,8 @@ export default class Task extends EventTarget {
   async run() {
     this.dispatchEvent(createBenchEvent('start', this));
     let totalTime = 0; // ms
+    let min = Infinity;
+    let max = -Infinity;
```

Comment:
if we want to get the min and max, we need to return them actually, and when you do, add it to the documentation.

Reply:
And aren't they in the wrong order? min should be -Infinity.

Reply:
> if we want to get the min and max, we need to return them actually, and when you do, add it to the documentation.

I do not understand what you're saying. There were min and max in the first place just a different way to get them.

> And aren't they in the wrong order? min should be -Infinity.

I think you may have misunderstood. If min is -Infinity it means that there is no value smaller than this min

Reply:
Oh I get it now.

Reply:
> I do not understand what you're saying. There were min and max in the first place just a different way to get them.
Yes, we used to get them by samples[samples.length] for instance, but since we do not sort now, we need to return them along other results like `mean`, `variance`, ...

Reply:
> > I do not understand what you're saying. There were min and max in the first place just a different way to get them.
> 
> Yes, we used to get them by samples[samples.length] for instance, but since we do not sort now, we need to return them along other results like `mean`, `variance`, ...

Oh, I got it. Looks like we still need to call `sort()` otherwise the result will be incorrect